packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -40,7 +40,7 @@ export interface PreferencesState extends BaseState {
   selectedAddress: string;
   useTokenDetection: boolean;
   useNftDetection: boolean;
-  openSeaEnabled: boolean;
+  displayNftMedia: boolean;
```

Comment:
It would help to explain which two things are changing as well: the `openSeaEnabled` => `displayNftMedia` rename, and the `setOpenSeaEnabled` => `setDisplayNftMedia` rename.

The current entry leaves out the method rename, and it describes the state property as a "variable" which may be misleading.

Reply:
Thank you so much for the review! Addressed the PR description!